### PR TITLE
Adding git add -u

### PR DESCRIPTION
**Pull request by markusthoemmes**
git add -u is a very common option in my opinion as it adds every change in a tracked file, which represents the top section of `git status`

### CODE REVIEW

**Comment by waldyrious** [comment]
Thanks, that's pretty handy indeed. Maybe also adjust the description of the `git add .` so the distinction is clearer?

**Comment by igorshubovych** [comment]
I don't think there should be text
- ... (top section of `git status` only)

Some users have different output of git status command, i.e. using [scmpuff](scmpuff).

**Comment by igorshubovych** [comment]
Also have you think about making it shorter?

E.g.
- Add changes only in already tracked files

Or anything like that.

**Comment by waldyrious** [comment]
I agree with @igorshubovych's suggestions :+1:

**Comment by markusthoemmes** [comment]
Good feedback, tried to incooperate your suggestions.

File: pages/common/git-add.md [hunk 1]
```diff
@@ -6,10 +6,14 @@
 
 `git add {{PATHSPEC}}`
 
-- Add all current changes to the index
+- Add all files (trackend and untracked)
```
Comment (waldyrious) [inline pages/common/git-add.md]: typo: "trackend" --> "tracked"

**Comment by markusthoemmes** [comment]
Good catch, fixed!

**Comment by igorshubovych** [comment]
LGTM,

@markusthoemmes could you please squash 3 commits into one?

**Comment by markusthoemmes** [comment]
Sure thing, all done!

**Comment by waldyrious** [comment]
Awesome, thanks!

**Comment by igorshubovych** [comment]
@marekhrabe 
Thank you for your contribution!

**Pull request merged**